package.json

```diff
@@ -66,9 +66,9 @@
     "@types/redux": "3.6.0",
     "@types/redux-localstorage": "1.0.8",
     "@types/rimraf": "2.0.3",
-    "@types/webdriverio": "4.13.3",
-    "@typescript-eslint/eslint-plugin": "2.12.0",
-    "@typescript-eslint/parser": "2.12.0",
+    "@types/webdriverio": "5.0.0",
```

Comment:
we need to leave `webdriverio` and `spectron` alone unless we're updating `electron` as the versions need to correspond. I added a note in the README but looking back on it now it doesn't seem so clear on that point.

Reply:
ahh, completely missed that readme update